build_image.sh

```diff
@@ -87,11 +87,24 @@ generate_onie_installer_image()
           $ONIE_INSTALLER_PAYLOAD
 }
 
+generate_onie_device_list()
+{
+    # Generate asic-specific ONIE device list
+    rm -rf ./installer/x86_64/devices/
+    mkdir -p ./installer/x86_64/devices/
```

Comment:
You just need to create an empty file in this folder. No need to remove directory.

Reply:
[generate_onie_installer_image](https://github.com/Azure/sonic-buildimage/blob/master/build_image.sh#L72)
I am following the above code to refresh the folder every time. And the folder needs to be created the first time.